README.md

```diff
@@ -1,8 +1,63 @@
 # Previous
-The bot for ~~annoying nextcord members~~ managing some of nextcord's unique tasks.
+The bot managing the Official [Nextcord Discord Server][NEXTCORDSERVER].
+
+# Features
+
+- **Autopaste:**
+    Automatically paste the contents of a file or codeblock to [our paste service][PASTESERVICE]. \
```

Comment:
Can you change it to `paste the content of a file` instead of `paste the contents of a file`. I might be wrong on this one TBH, but I'm not sure :joy:.